src/components/MenuItemCard.tsx

```diff
@@ -22,7 +22,9 @@ export const MenuItemCard: FunctionComponent<MenuCardProps> = ({
     <MenuItem w="95%" onClick={onClick} data-testid={testId}>
       <Card padding="1">
         <HStack spacing="2">
-          <Text variant={textColor}>{icon}</Text>
+          <Text variant={textColor} as="div">
+            {icon}
+          </Text>
```

Comment:
can we use <Box/> instead, and i think you dont need any textColor for the icon?

Reply:
have changed to use Box instead which output the simple div, and test in UI and didn't affect anything.